app/src/main/java/com/pokerio/app/MyFirebaseMessagingService.kt

```diff
@@ -0,0 +1,23 @@
+package com.pokerio.app
+
+import android.util.Log
+import com.google.firebase.messaging.FirebaseMessagingService
+import com.google.firebase.messaging.RemoteMessage
+
+class MyFirebaseMessagingService : FirebaseMessagingService() {
```

Comment:
Please make the class name more unique (ex. `PokerioFirebaseMessagingService`)

Reply:
DONE